app/code/Magebit/PageListWidget/etc/module.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
```

Comment:
Add dockblock

Reply:
All files should contane the same dockblock. 
You already used one in PageList.php

`/**
 * @copyright Copyright (c) 2024 Magebit (https://magebit.com)
 * @author    Magebit <info@magebit.com>
 * @license   GNU General Public License ("GPL") v3.0
 */`